i18naddress/downloader.py

```diff
@@ -0,0 +1,91 @@
+import io
+import json
+import logging
+from multiprocessing.pool import ThreadPool
+from multiprocessing import JoinableQueue, Manager
+import os
+
+import requests
+from retrying import retry
+
+from . import COUNTRIES_VALIDATION_DATA_DIR
+
+MAIN_URL = 'http://i18napis.appspot.com/address/data'
+COUNTRY_PATH = os.path.join(COUNTRIES_VALIDATION_DATA_DIR, '%s.json')
+
+logger = logging.getLogger(__name__)
+work_queue = JoinableQueue()
+manager = Manager()
+
+
+@retry
+def fetch(url):
+    logger.debug(url)
+    data = requests.get(url).json()
+    return data
+
+
+def process(key):
+    url = '%s/%s' % (MAIN_URL, key)
+    data = fetch(url)
+    for unwanted in [
```

Comment:
Please remove this filtering and regenerate the JSON files so they are complete.